installer/mac/tools/renamesha256.rb

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env ruby
```

Comment:
This line makes me hesitate a little bit only because it'd be nice not to introduce too many dependencies for building Chain Core, and ruby is a pretty significant new dep here. (As far as I know it's new.)

This ruby script in particular pretty much just calls out to the shell, so maybe it would be clearer as a shell script?

```bash
#!/bin/bash
set -eo pipefail
digest=`shasum -a 256 "$1"|cut -b-64`
mv "$1" "$1"-$digest
```